test/inductor/test_torchinductor.py

```diff
@@ -787,6 +787,79 @@ def repeat(x, n):
         self.assertEqual(expect, actual)
         self.assertEqual(actual, repeat(x, 3))
 
+    @skipIfRocm
+    def test_neg_index(self):
+        def test(fn, inps, has_assert: bool, has_wrapping=True):
+            for dynamic in (True, False):
+                fn_opt = torch.compile(dynamic=dynamic)(fn)
+                if self.device == "cpu":
+                    code = run_and_get_cpp_code(fn_opt, *inps)
+                    lines = code.split("\n")
+                    found = False
+                    for line in lines:
+                        # Line will look something like this:
+                        # TORCH_CHECK((0 <= tmp5) & (tmp5 < ks1), error_msg)
```

Comment:
You don't want to match this one, you want to match op.where, that is, the ternary operator when it's not vectorised and `blendv` when it is.

Reply:
Is `blendv` a c++ function? I don't see it in any of the C++ generated code

Reply:
It's an [intel intrinsic](https://www.intel.com/content/www/us/en/docs/cpp-compiler/developer-guide-reference/2021-8/mm256-blendv-epi8.html). It's generated if the code is vectorised (it may not be the case in that test, idk).